src/Storage/Storage.Management/help/Get-AzStorageBlob.md

```diff
@@ -93,7 +93,7 @@ PS C:\> do
      if($Blobs.Length -le 0) { Break;}
      $Token = $Blobs[$blobs.Count -1].ContinuationToken;
  }
- While ($Token -ne $Null)
+ While ($null -ne $Token)
```

Comment:
Please indicate why need this change.

Reply:
FYI, https://github.com/PowerShell/PSScriptAnalyzer/issues/1021

Reply:
Good to know it. Thanks!